tests/population/conftest.py

```diff
@@ -1,6 +1,72 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
 import pytest
 
 
+@pytest.fixture
```

Comment:
The fixtures in this file were just copied over from test_base_population.py so they can also be used to instantiate the base population in mortality test.